ui/components/MeshModelComponent.js

```diff
@@ -326,6 +326,28 @@ const MeshModelComponent = ({ view, classes }) => {
       },
 
     },
+    {
+      name : "registrant",
+      label : "Registrant",
+      options : {
+        sort : view === COMPONENTS || view === RELATIONSHIPS ? true : false,
```

Comment:
```suggestion
        sort : view === COMPONENTS || view === RELATIONSHIPS ,
```
the ternary check is not required